### PR TITLE
[FEATURE1] First Feature Implemented

### PR DESCRIPTION
**Pull request by sudo-pacman-syu**
Following changes were made:

- New feature.txt was added
- feature1 folder was created

### CODE REVIEW

File: new-feature1.txt [hunk 1]
```diff
@@ -1 +1,3 @@
-This file was created in the feature1 branch
\ No newline at end of file
+This file was created in the feature1 branch
+
+One more change in the original commit
```
Comment (srshah27) [inline new-feature1.txt:3]: Hmmm, great change

**Review by srshah27 — APPROVED** [review]

**Review by sudo-pacman-syu — COMMENTED** [review]
idk man sounds fun

File: new-feature1.txt [hunk 2]
```diff
@@ -1 +1,3 @@
-This file was created in the feature1 branch
```
Comment (sudo-pacman-syu) [inline new-feature1.txt]: Is this right?

**Review by srshah27 — CHANGES_REQUESTED** [review]
Reply to this and I will approve

**Review by srshah27 — APPROVED** [review]

**Pull request merged**